vault/external_tests/raft/raft_test.go

```diff
@@ -40,10 +41,15 @@ type RaftClusterOpts struct {
 	InmemCluster                   bool
 	EnableAutopilot                bool
 	PhysicalFactoryConfig          map[string]interface{}
+	PhysicalFactory                func(t testingintf.T, coreIdx int, logger log.Logger, conf map[string]interface{}) *vault.PhysicalBackendBundle
```

Comment:
Are you using this?

Reply:
I think I used it in an earlier iteration of the ent tests and forgot to remove it.